centreon-open-tickets/widgets/open-tickets/src/index.php

```diff
@@ -146,7 +146,7 @@
 
     FROM hosts h
     LEFT JOIN customvariables cv5 ON (
-        h.host_id = cv5.host_id AND cv5.service_id IS NULL AND cv5.name = '" . $macro_tickets['ticket_id'] . "'
+        h.host_id = cv5.host_id AND (cv5.service_id IS NULL OR cv5.service_id = 0) AND cv5.name = '" . $macro_tickets['ticket_id'] . "'
```

Comment:
I am wondering if there is a performance gap betweeen the `OR` and a [`ISNULL`](https://dev.mysql.com/doc/refman/8.0/en/comparison-operators.html#function_isnull) /  [`COALESCE`](https://dev.mysql.com/doc/refman/8.0/en/comparison-operators.html#function_coalesce) in the JOIN 🤔
```suggestion
        h.host_id = cv5.host_id AND ISNULL(cv5.service_id, 0) = 0 AND cv5.name = '" . $macro_tickets['ticket_id'] . "'
```

Reply:
Sorry it is a backport, I was not picked up as a reviewer in the main PR.
Feel free to Resolve this conversation ✅ 
It was just a thought ¯\_(ツ)_/¯